src/inspect_ai/_view/www/src/api/Types.mjs

```diff
@@ -35,11 +35,21 @@
  */
 
 /**
-* @typedef {Object} Capabilities
-* @property {boolean} downloadFiles - Indicates if file downloads are supported.
-* @property {boolean} webWorkers - Indicates if web workers are supported.
-*
+ * Fields shared by EvalSample and SampleSummary.
+ * Contains only fields that are copied verbatim in src/inspect_ai/log/_recorders/eval.py.
+ *
+ * @typedef {Object} SampleInterface
```

Comment:
It's true that we don't need anything but the scores at this point. But I thought: if we are doing something like this, we might as well create a type that stores all fields that are common between `EvalSample` and `SampleSummary`. Then, if a function accepts this type, it can freely take either of those. This is why the word “Interface” jumped to mind. But I agree that in this context it could be confusing. How about `BasicSampleData`?

Reply:
Agree - I think this is nice and actually ran into a place where I wanted to use this type!